src/main/java/controller/StudentsList.java

```diff
@@ -0,0 +1,35 @@
+package controller;
+
+import model.Student;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/3/2019.
+ */
+class StudentsList {
+
+    static List<Student> students = init();
+
+    private static List<Student> init() {
```

Comment:
can you work with students without initializing it? I believe not. This method is the candidate for static block

Reply:
done - remove redundant students attribute

Reply:
done - init is called in constructor